tutoraspects/commands_v1.py

```diff
@@ -169,12 +169,12 @@ def init_clickhouse() -> list[tuple[str, str]]:
     "--fail_on_error", is_flag=True, default=False, help="Allow errors to fail the run."
 )
 def performance_metrics(
-    course_key, dashboard_slug, slice_name, print_sql, fail_on_error
+    course_name, dashboard_slug, slice_name, print_sql, fail_on_error
```

Comment:
you'll probably need org too if you use course name since course name isn't unique